job-service-acceptance-tests/src/integration-test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -130,11 +130,6 @@ public void testSetup() throws Exception {
         testItemAssetIds = generateWorkerBatch();
     }
 
-//    @Test
-//    public void testJobCompletionWithTaskDataObject() throws Exception {
-//        testJobCompletion(true);
-//    }
-
```

Comment:
I did not previously comment on this section so why are you now deleting this? Is this something that should be enabled in future?

Reply:
Added it back, was there before. So not sure if its going to be used.